database/src/postgres/migrations/2023-12-07-144820_epochs/up.sql

```diff
@@ -0,0 +1,20 @@
+CREATE TABLE IF NOT EXISTS validators (
+    epoch_id text NOT NULL,
+    epoch_height numeric(20,0) NOT NULL,
+    epoch_start_height numeric(20,0) NOT NULL,
+    validators_info jsonb NOT NULL
+);
+
+ALTER TABLE ONLY validators
+    ADD CONSTRAINT validators_pk PRIMARY KEY (epoch_id, epoch_start_height);
```

Comment:
The first 2 epochs in the network have the same epoch_id. But I researched the epochs and found out that there is no sense to index the genesis epoch because it does not have any important data. I refused to index the genesis epoch and wanted to change the composite primary key to epoch_id but forgot about it. Thanks for pointing that! I will make the change.